src/login7-payload.js

```diff
@@ -222,6 +272,16 @@ module.exports = class Login7Payload {
     return variableData.offset += value.length * 2;
   }
 
+  addVariableDataInt(variableData, value) {
+    value || (value = '');
+    variableData.offsetsAndLengths.writeUInt16LE(variableData.offset);
+    variableData.offsetsAndLengths.writeUInt16LE(4);
+    //placeHolder
+    this.tempHolder = variableData.data.getPos();
```

Comment:
Can you change `tempHolder` into a more meaningful variable name? Like tempHolder of what?

Reply:
Now that i look at this again, i guess it can be better designed, it doesn't make sense that addVariableDataInt write the placeholder in this, it should return it or there should be another func..